Goobi/install/config/goobi_metadataDisplayRules.xml

```diff
@@ -17,6 +17,18 @@
             <textarea ref="Description">
                 <label></label>
             </textarea>
+
+            <!-- Restrict to specific dataset, try to search for e.g Per -->
+            <kulturnav ref="person">
+                <source>entity.dataset:e0c682f9-a594-4fe1-8c01-f208f52d79ac</source>
+                <field></field>
+            </kulturnav>
+
+            <!-- Get place of publication from KulturNav -->
+            <kulturnav ref="PlaceOfPublication">
+                <source>entityType:Place</source>
+                <field></field>
+            </kulturnav>
```

Comment:
I would also prefer a basic configuration in the docker image

Reply:
I didn't get here on the basic configuration in the docker image. My understanding is that docker is copying everything from `Goobi/install/config` when building new image in which, this particular config will always be copied?

Reply:
Yes, it is copied to the docker image. So the person and PlaceOfPublication metadata will be a "kulturnav field" for the basic docker image. I would prefer to have these lines removed from the configuration, so we would have a basic configuration that just shows input fields for the metadata.

Reply:
This commit See https://github.com/intranda/goobi-workflow/pull/23/commits/475d40cd80d8c2259e89c634911064c4b01fadfd
removes the lines to have a basic configuration.